.buildkite/pipeline.yml

```diff
@@ -39,3 +39,30 @@ steps:
     depends_on: "build"
     agents:
       os: "macOS"
+
+  #################
+  # UI Tests
+  #################
+  - label: "🧪 UI Tests (iPhone)"
+    command: |
+      buildkite-agent artifact download build-products.tar .
+      tar -xf build-products.tar
+      bundle install
+      bundle exec pod install
+      rake mocks &
+      bundle exec fastlane test_without_building name:WordPressUITests try_count:3 device:"iPhone 11" ios-version:"14.1"
+    depends_on: "build"
+    agents:
+      os: "macOS"
```

Comment:
Same as above – possibly not, but I've had a couple different times where this happened.

Not sure if we want to specify something globally, or make each step specify its own runner agent (because not everything may want to run on macOS)